frontend/src/js/query-node-editor/ConceptDropzone.tsx

```diff
@@ -17,23 +21,35 @@ interface PropsT {
   onDropConcept: (concept: DragItemConceptTreeNode) => void;
 }
 
+export const droppableObjectIsConceptTreeNode = (
+  node: PossibleDroppableObject,
+): node is DragItemConceptTreeNode => {
+  return node.type === DNDType.CONCEPT_TREE_NODE;
+};
+
+export const canDropConceptTreeNodeBeDropped = (node: StandardQueryNodeT) => {
+  return (item: PossibleDroppableObject) => {
```

Comment:
Good idea to share the code. Could you simplify this a little?

- rename to `canNodeBeDropped`
- move the code to `src/js/model/node.ts`
- maybe even just use two parameters `(node, item)` instead of returning a function and inline the function where it's called?

Reply:
> maybe even just use two parameters (node, item) instead of returning a function and inline the function where it's called?
I thought that it might be a little simpler , but with two params looks better so changed it to that